orbitx/physics_entity.py

```diff
@@ -60,6 +60,12 @@ def v(self):
     def v(self, x):
         self.proto.vx = x[0]
         self.proto.vy = x[1]
+        
+        
+    #temporary solution to detect AYSE, need improvement
```

Comment:
did we decide on what the improvement for this should be? a field in the protobuf? or just whether the the entity's name is AYSE?